packages/wallets/src/features/cfd/screens/MT5TradeScreen/MT5TradeLink/mt5-mobile-redirect-option.scss

```diff
@@ -0,0 +1,32 @@
+.wallets-mobile-redirect-option {
+    display: flex;
+    flex-direction: column;
+    width: 100%;
+    padding: 2.4rem;
+    gap: 1rem;
+
+    &__button,
+    a {
+        border: none;
+        display: flex;
+        justify-content: space-between;
+        align-items: center;
+        text-decoration: none;
+        outline: none;
+        height: 3.4rem;
+        padding: 0.8rem;
+        border-radius: 0.8rem;
+        width: 100%;
+        gap: 0.5rem;
+        background-color: var(--general-section-4);
+
+        &--blue {
+            background-color: #0364b9;
```

Comment:
can you replace this hex color with an css variable (if it exist)

Reply:
nope 😓 no exist

Reply:
nope 😓 no var for this hex exist

and I don't think this colour is being used anywhere else other than the icons... so I don't think we need a color variable for it